packages/electrode-archetype-react-app-dev/package.json

```diff
@@ -46,7 +46,7 @@
     "electrode-electrify": "^1.0.0",
     "electrode-gulp-helper": "^1.0.5",
     "electrode-webpack-reporter": "^0.2.4",
-    "eslint": "^3.9.1",
+    "eslint": "^3.18.0",
```

Comment:
please undo this change.  we want to keep the changes focus on the archetype component only.